exercises/largest-series-product/src/Series.hs

```diff
@@ -1,4 +1,4 @@
 module Series (largestProduct) where
 
 largestProduct :: Int -> String -> Maybe Integer
-largestProduct = error "You need to implement this function."
+largestProduct series digits = error "You need to implement this function."
```

Comment:
`series` seems confusing to me since an integer is not really a series. It is the size of the seires we are considering within the digits. I would think either `size` or `span`.